tests/resources.tf

```diff
@@ -42,10 +42,32 @@ resource "opslevel_infrastructure" "big_infra" {
 
 # Rubric Category resources
 
-resource "opslevel_rubric_category" "mock_category" {
+resource "opslevel_scorecard" "small" {
   name = "Mock Category"
 }
 
+# Scorecard resources
+
+resource "opslevel_scorecard" "big" {
```

Comment:
This should be `opslevel_service` same with the other .hcl files and the file names.

Reply:
Good catch. I'm still updating the tests too